mappings/net/minecraft/world/gen/chunk/ChunkGeneratorType.mapping

```diff
@@ -2,16 +2,31 @@ CLASS net/minecraft/class_5284 net/minecraft/world/gen/chunk/ChunkGeneratorType
 	FIELD field_24514 defaultBlock Lnet/minecraft/class_2680;
 	FIELD field_24515 defaultFluid Lnet/minecraft/class_2680;
 	FIELD field_24516 config Lnet/minecraft/class_5311;
+	FIELD field_24780 OBJECT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24781 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_24782 noiseConfig Lnet/minecraft/class_5309;
 	FIELD field_24783 bedrockCeilingY I
 	FIELD field_24784 bedrockFloorY I
+	FIELD field_24785 seaLevel I
+	FIELD field_24786 disabledMobGeneration Z
```

Comment:
maybe mob spawn than generation, so `noMobSpawn`?

Reply:
mmh, the string is `"disable_mob_generation"`